docs-ref-services/preview/mgmt-devtestlabs-readme.md

```diff
@@ -1,12 +1,13 @@
 ---
-title: 
+title: DevTest Labs Azure SDK for Python
+description: Learn how to manage Azure DevTest Labs resources with the Azure SDK for Python.
 keywords: Azure, python, SDK, API, azure-mgmt-devtestlabs, devtestlabs
 author: lmazuel
 ms.author: lmazuel
 ms.date: 10/31/2022
 ms.topic: reference
 ms.devlang: python
-ms.service: devtestlabs
+ms.service: devtest-lab
```

Comment:
This is a package-level overview (filename ends in -readme.md) and its metadata is generated by automation. This metadata will be overwritten on the next release of this package. To make this change persist you also need to open a PR to the metadata CSV file here: https://github.com/Azure/azure-sdk/blob/main/_data/releases/latest/dotnet-packages.csv

Set the value of MSDocService -- https://dev.azure.com/azure-sdk/internal/_wiki/wikis/internal.wiki/602/Package-Onboarding-step-by-step?anchor=docs-metadata

Reply:
@danieljurek Should I update the corresponding packages file for Python in this case?

Reply:
Yes, the CSV link should've been: https://github.com/Azure/azure-sdk/blob/main/_data/releases/latest/python-packages.csv

Reply:
Tagged you in the PR. Thanks a lot for your support!